### PR TITLE
fix: update email settings

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

**Review by Bouncey — CHANGES_REQUESTED** [review]
There is an issue where, if I change my email then visit `/` without verifying, I get redirected to `/update-email` asking me to verify my old email

The UI of the email form doesn't make any sense now that I am looking at it. I will add a picture with some improvements.

**Comment by Bouncey** [comment]
Current Email should be in a `<p>` or `<span>`, or something just as uneditable. 

email

**Comment by raisedadead** [comment]
The other issue will be resolved with https://github.com/freeCodeCamp/freeCodeCamp/pull/17669

**Comment by raisedadead** [comment]
Regarding the UI. I think I am not an expert at the moment. But will give it a quick try. I just added back the original views that we commented previously.

File: common/app/routes/Settings/components/Email-Settings.jsx [hunk 1]
```diff
@@ -139,8 +124,12 @@ class EmailSettings extends PureComponent {
             </HelpBlock>
           </FullWidthRow>
         }
-
-
+        <FullWidthRow>
+          <EmailForm
```
Comment (Bouncey) [inline common/app/routes/Settings/components/Email-Settings.jsx]:
```jsx
<EmailForm
  email={ email }
  initialValues={{ email, confirmEmail: '' }}
/>
```

File: common/app/routes/Settings/components/EmailForm.jsx [hunk 2]
```diff
@@ -79,10 +79,20 @@ class EmailForm extends PureComponent {
 
     return (
       <form id='email-form' onSubmit={ handleSubmit(this.handleSubmit) }>
+        <Row className='inline-form-field'>
+          <Col sm={ 3 } xs={ 12 }>
+            Current Email
+          </Col>
+          <Col sm={ 9 } xs={ 12 }>
+            <h4>
+              { email.value }
```
Comment (Bouncey) [inline common/app/routes/Settings/components/EmailForm.jsx]: `{ this.props.email }`

Unless you want to de-structure it at Line 72

**Review by Bouncey — CHANGES_REQUESTED** [review]
Updating my email changes my current email on the page. 

I have provided setups to fix this :+1:

**Review by Bouncey — APPROVED** [review]

**Pull request merged**